src/grt/src/RoutingTracks.h

```diff
@@ -89,6 +89,9 @@ class RoutingTracks
   int getLocation() const { return location_; }
   int getNumTracks() const { return num_tracks_; }
   bool getOrientation() const { return orientation_; }
+
+  const static bool HORIZONTAL = 0;
+  const static bool VERTICAL = 1;
```

Comment:
Are these still needed now that you are using opendb layers?

Reply:
I use them in some places where the function expects bools as arguments. I will update the function parameters.